apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-a-beneficiaries.txt

```diff
@@ -1,2 +1,2 @@
+INSERT|567834|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|543217066U|Doe|John|A|3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|1|1|1|1|1|1|1|1|1|1|0|0|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|0|0|003|003|003|003|003|003|003|003|003|003|||00|00|00|00|00|00|00|00|00|00|||S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|0|003|003|003|003|003|003|003|003|003|003|003||000|000|000|000|000|000|000|000|000|000|000||Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|N|**|**|**|**|**|**|**|**|**|**|**|**|00|00|01|01|05|06|07|08|01|02|00|00|204 SOUTH ST|7560 123TH ST|SURREY|DAEJEON SI 34867|COLOMBIA|SURREY|BALTIMORE|MD|549081955|30-JUL-2020||18325
```

Comment:
Please don't use realistic-looking data here. I know it's obvious to you that it's just your home address (or whatever it is), but you're setting up some future developer or security person to have a panic attack.

For addresses, I like "123 MAIN ST, APT 456, PODUNK, IA, 12345" or something along those lines. Visually, it's obvious to anyone that it didn't come from real data.

Reply:
yes sir